robots/flakefinder/report.go

```diff
@@ -348,6 +349,67 @@ func Report(results []*Result, reportOutputWriter *storage.Writer) error {
 	return nil
 }
 
+// SortTestsByRelevance sorts given tests according to the severity from the test data, where tests with a higher
+// severity have a smaller index in the slice than tests with a lower severity.
+// The returned slice does not contain
+// duplicates, thus if a test has data with several severities the highest one is picked, leading to an earlier
+// encounter in the slice.
+func SortTestsByRelevance(data map[string]map[string]*Details, tests []string) (testsSortedByRelevance []string) {
+	foundTests := map[string]bool{}
+	for _, test := range tests {
```

Comment:
I think you don't need this, in case of miss the map returns the 0 value of the record which is false.

Reply:
Good find!

```

```